src/main/java/pro/trevor/tankgame/state/board/floor/HealthPool.java

```diff
@@ -0,0 +1,32 @@
+package pro.trevor.tankgame.state.board.floor;
```

Comment:
You might want to make a constructor that accepts JSON to create a HealthPool

Reply:
ACK